### ISSUE TITLE
Create a Git repository with the code

### ISSUE THREAD

**Issue by Hneuschmidt** [StefanJAuer/RaySAR#11]
The code is currently only available as a bundled release. It would be advantageous to have the code be available
also as a git repository.

Concrete advantages of a having repository:
- The code will be under version control, making it easier to add incremental changes and roll back in cases of bugs
- It makes it possible to add changes and improvements that are much smaller that what would make sense for a new release
- Experimental features can be added on a separate branch, without compromising the status of the main project
- Bug fixes and improvements by third parties can be integrated via pull request if desired

My first step will be to add all the code in its current form to a branch.
For now, I will leave out files that are not text files, which generally do not benefit from being in a git repository.
This specifically includes the compiled version of the matlab part and POV-Ray, example models and the pdf version of the documentation.
These files will still be available through the releases mechanism as is the case right now. 
I think the documentation should also live in the repo in a source version that can be compiled by users or hosted somewhere,
but that is a for a separate issue.

In general, I plan to use these issues as a form to communicate what I am working on, so @StefanJAuer can weigh in
if there is something to discuss and as a way to document what is being worked on and the reasoning behind changes.

Once a project is complete, I will close the issue.
